core/docker/build.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+usage() {
+    cat <<EOF 1>&2
+Usage: $0 [-h] [-a <ARCHITECTURES>] [-r <REVISION>]
+Builds the Trino Docker image
+
+-h       Display help
+-a       Build the specified comma-separated architectures, defaults to amd64,arm64
+-r       Build the specified Trino revision (version), downloads all required artifacts
```

Comment:
revision -> release

(to me, "revision" sounds svn-ish)

Reply:
How about `release version` then?